docs/doc/15-sql-functions/61-ai-functions/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'AI Functions'
+---
+
+Databend can utilize the OpenAI `Code-Davinci-002` engine to translate natural language into SQL queries.
+
+By integrating OLAP and AI, Databend simplifies the process of writing SQL queries based on your table schema.
+
+The `ai_to_sql` function allows us to effortlessly generate SQL queries using natural language.
+
+## Syntax
+
+```sql
+USE <your-database>;
+SELECT * FROM ai_to_sql('<prompt>', '<openai-api-key>');
+```
+
+## Example
+
+:::note
+Please note that the generated SQL query may need to be adapted to match Databend's syntax and functionality, as it might be based on PostgreSQL-standard SQL.
+:::
+
+
+```sql
+CREATE DATABASE openai;
+USE openai;
+
+CREATE TABLE users(
+    id INT,
+    name VARCHAR,
+    age INT
+);
+
+SELECT * FROM ai_to_sql('List all users older than 30 years', '<openai-api-key>');
```

Comment:
`openai-api-key` could be a setting or a config?

It's not safe to pass the key as a function argument.

Reply:
Yes, we can do it later, for example: if the ai_to_sql argument only one, we can use the key from our config.  
The key can be revoked and re-create on openai platform.